### PR TITLE
Setup Drizzle + Docker postgresql

### PR DESCRIPTION
**Pull request by andresgutgon**
# What?
☝️ That

### CODE REVIEW

File: apps/web/drizzle/0000_talented_purple_man.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+CREATE TABLE IF NOT EXISTS "prompts" (
+	"created_at" timestamp DEFAULT now() NOT NULL,
+	"updated_at" timestamp DEFAULT now() NOT NULL
+);
```
Comment (andresgutgon) [inline apps/web/drizzle/0000_talented_purple_man.sql:4]: Dumb version in order to merge this first

File: apps/web/src/db/schema/schemaHelpers.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+      .$onUpdateFn(() => new Date()),
+  }
+}
+
+/**
+ * Case-insensitive column for use in unique indexes.
+ */
+export function lowercaseColumn(column: AnyPgColumn): SQL {
+  return sql`lower(${column})`
+}
```
Comment (geclos) [inline apps/web/src/db/schema/schemaHelpers.ts:19]: what's this for

Comment (andresgutgon) [reply apps/web/src/db/schema/schemaHelpers.ts:19]: Copy & paste. I'll remove

File: tools/typescript/package.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@latitude-llm/typescript",
+  "name": "@latitude-data/typescript-config",
```
Comment (geclos) [inline tools/typescript/package.json:2]: can't we keep continue using @latitude/whatever?

**Pull request merged**